src/shared/components/challenge-detail/Specification/SideBar/index.jsx

```diff
@@ -137,6 +137,16 @@ export default function SideBar({
             </div>
           )
         }
+        <div>
+          <h2>
+            LEARN:
+          </h2>
+          <p styleName="link-like-paragraph">
+            <a href="https://www.topcoder.com/thrive/articles/all-about-topcoder-challenges-tasks-and-gig-work-opportunities">
```

Comment:
@dedywahyudi we need to use config var to point this link to right env, like Dev, Stag, etc.

ex. `config.URL.COMMUNITY`